package.json

```diff
@@ -29,6 +29,9 @@
     },
     "devDependencies": {
         "babel-jest": "^23.6.0",
+        "babel-plugin-add-module-exports": "^1.0.2",
```

Comment:
Do you need these babel plugins and preset since you ignore transformation of Picker for Jest?